docs/book/common_code/machineset_controller.md

```diff
@@ -19,13 +19,6 @@ which implement their intent by modifying provider-specific `Cluster` and
 [import:'MachineSetSpec'](../../../pkg/apis/cluster/v1alpha1/machineset_types.go)
 {% endmethod %}
 
-{% method %}
-## MachineSetTemplateSpec
-
-{% sample lang="go" %}
-[import:'MachineSetTemplateSpec'](../../../pkg/apis/cluster/v1alpha1/machineset_types.go)
-{% endmethod %}
-
```

Comment:
I don't think this should be removed, rather it should reference `MachineTemplateSpec`: https://github.com/kubernetes-sigs/cluster-api/blob/master/pkg/apis/cluster/v1alpha1/machineset_types.go#L75-L87

Reply:
Thanks @detiber , I found this was actually fixed in https://github.com/kubernetes-sigs/cluster-api/pull/664/files